src/metabase/models/card.clj

```diff
@@ -1051,7 +1051,9 @@
                   :mr         [:model/ModerationReview [:and
                                                         [:= :mr.moderated_item_type "card"]
                                                         [:= :mr.moderated_item_id :this.id]
-                                                        [:= :mr.most_recent true]]]}})
+                                                        [:= :mr.most_recent true]]]
+                  ;; workaround for dataflow :((((((
```

Comment:
I thought about walking the field expressions looking for sub-selects and automatically adding these joins, but the problem is that we need to know the model, the table name isn't good enough.